fortio_main.go

```diff
@@ -272,14 +272,14 @@ func fortioLoad(justCurl bool, percList []float64) {
 		log.LogVf("Generated Labels: %s", labels)
 	}
 	ro := periodic.RunnerOptions{
-		QPS:         qps,
-		Duration:    *durationFlag,
-		NumThreads:  *numThreadsFlag,
-		Percentiles: percList,
-		Resolution:  *resolutionFlag,
-		Out:         out,
-		Labels:      labels,
-		Exactly:     *exactlyFlag,
+		QPS:           qps,
+		Duration:      *durationFlag,
+		NumThreads:    *numThreadsFlag,
+		Percentiles:   percList,
+		Resolution:    *resolutionFlag,
+		Out:           out,
+		Labels:        labels,
+		Exactly:       *exactlyFlag,
 		RequestJitter: *requestJitterFlag,
```

Comment:
alternative is to call this just "jitter" without request and it would then not reformat/make the diff smaller (also easier to type `-jitter`)

Reply:
good idea. It would make it more concise. I was thinking that different types of jitters might be added later.

Reply:
yes it's possible (we add more later, or for instance split the connection jitter from the request jitter) but for now let's have one

I'm also wondering if it should be configurable in % or as a float but a boolean is fine for now I think (also wondering a bit about the int calculation, I guess it's in nanoseconds so probably fine)